.github/workflows/build.yml

```diff
@@ -244,26 +250,23 @@ jobs:
       - name: Download Build Artifacts
         uses: actions/download-artifact@v2
 
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2.2.4
+        with:
+          version: ${{ env.PNPM_VERSION }}
+        
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
+          cache: "pnpm"
           node-version: ${{ env.NODE_VERSION }}
 
-      - name: Setup NPM
-        run: npm install -g npm@${{ env.NPM_VERSION }}
-
       - name: Install Dependencies
-        uses: bahmutov/npm-install@v1
-        with:
-          working-directory: |
-            tools/hangar
-            examples/tests/valid
-            examples/tests/invalid
-          install-command: npm ci
+        run: pnpm install --ignore-scripts --frozen-lockfile --filter hangar --filter examples-valid --filter examples-invalid
```

Comment:
Is there still a reason to use `--ignore-scripts`?

Reply:
Yeah, to skip the scripts at the root of the repo (mostly the setup_wasi.sh thing)